src/components/managers/products/addProduct/mediaSelectorFolder/thumbnailPicker.tsx

```diff
@@ -2,7 +2,7 @@ import { deleteFiles, getAllUploadedFiles } from 'api/admin';
 import { common_Media, common_ProductNew } from 'api/proto-http/admin';
 import React, { FC, useEffect, useState } from 'react';
 import styles from 'styles/thumbnailSelector.scss';
-import { DragDrop } from '../dragDrop';
```

Comment:
I assume this file will be removed ? Cuz we agreed that dragAndDrop is used only in media picker